openapi/azure.openai.deployment/original-openapi.yaml

```diff
@@ -0,0 +1,1929 @@
+{
```

Comment:
Do we have two openapi.yaml specs? I think only the one used with the connector is needed.

Reply:
That is the practice they have used for connectors with modified yamls, is to have the original yaml too.

Reply:
Ack